mne/selection.py

```diff
@@ -114,3 +122,75 @@ def read_selection(name, fname=None, info=None, verbose=None):
     if spacing == 'new':  # "new" or "old" by now, "old" is default
         sel = [s.replace('MEG ', 'MEG') for s in sel]
     return sel
+
+
+def _divide_to_regions(raw, add_stim=True):
```

Comment:
Oh this has EEG channels -- but the above goes for the "selection" option. Here it's the "position" option, which I would say the same thing about for non-data (`stim`, `chpi`, `eog`, etc.) channels.